Doc/library/ftplib.rst

```diff
@@ -201,14 +201,27 @@ FTP objects
 
    .. method:: FTP.retrbinary(cmd, callback, blocksize=8192, rest=None)
 
-      Retrieve a file in binary transfer mode.  *cmd* should be an appropriate
-      ``RETR`` command: ``'RETR filename'``. The *callback* function is called for
-      each block of data received, with a single bytes argument giving the data
-      block. The optional *blocksize* argument specifies the maximum chunk size to
-      read on the low-level socket object created to do the actual transfer (which
-      will also be the largest size of the data blocks passed to *callback*).  A
-      reasonable default is chosen. *rest* means the same thing as in the
-      :meth:`transfercmd` method.
+      Retrieve a file in binary transfer mode.
+
+      :param str cmd:
+        An appropriate ``STOR`` command: :samp:`"STOR {filename}"`.
+
+      :param callback:
+         A single parameter callable that is called
+         for each block of data received,
+         with its single argument being the data as :class:`bytes`.
+      :type callback: :term:`callable`
+
+      :param int blocksize:
+         The maximum chunk size to read on the low-level
+         :class:`~socket.socket` object created to do the actual transfer.
+         This also corresponds to the largest size of data
+         that will be passed to *callback*.
+         Defaults to 8192.
+
+      :param int rest:
+         A ``REST`` command to be sent to the server.
+         See the corresponding named parameter to the :meth:`transfercmd` method.
```

Comment:
The wording feels a little bit off-kilter. Perhaps a native speaker could help me improve this.